.vscode/settings.json

```diff
@@ -8,5 +8,16 @@
   "typescript.tsdk": "node_modules/typescript/lib",
   "[typescript]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
-  }
+  },
```

Comment:
The following config is suggested by VS Code MDX [doc](https://marketplace.visualstudio.com/items?itemName=unifiedjs.vscode-mdx).

Reply:
Continue the conversation with @calebpollman https://github.com/aws-amplify/amplify-ui/pull/3369#discussion_r1095173935

Reply:
Do we need to use a vscode extension in order to lint mdx files?

Reply:
We don't _have to_, but it'd be a lot better DX well developing so that developer don't have to wait until commit or a build failure